### DISCUSSION TITLE
Detect sound in vc

### DISCUSSION THREAD

**Discussion by IhsFace** [Rapptz/discord.py#8311] [Q&A]
Is there a way to detect sound in a voice channel? I want to play a file everytime the bot detects sound, is that possible?

**Comment by justanotherbyte** [comment]
Unfortunately discord.py does not provide a way to receive voice packets, so you'll need to figure out a different way to do this.

Maybe you could use the `on_voice_state_update` event? This wouldn't tell you when someone starts speaking, but rather when someone joins/leaves a voice channel, is muted or is deafened.
